src/types/tracker.ts

```diff
@@ -96,3 +141,71 @@ export type AnalyticsPayload = {
     sdk_token?: string
   }
 }
+
+export type AnalyticsTrackedEventNames =
+  | 'COMPLETE'
+  | 'COMPLETE_CROSS_DEVICE_MOBILE_SUCCESS'
+  | 'COUNTRY_SELECTION'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
+  | 'CROSS_DEVICE_CLIENT_INTRO'
+  | 'CROSS_DEVICE_DESKTOP_SUBMIT'
+  | 'CROSS_DEVICE_GET_LINK'
+  | 'CROSS_DEVICE_INTRO'
+  | 'CROSS_DEVICE_LINK_METHOD_SELECTED'
+  | 'CROSS_DEVICE_LINK_METHOD_SELECTED'
+  | 'CROSS_DEVICE_LINK_METHOD_SELECTED'
+  | 'CROSS_DEVICE_MOBILE_NOTIFICATION_SENT'
+  | 'CROSS_DEVICE_MOBILE_SUBMIT'
+  | 'CROSS_DEVICE_SMS_FAILED'
+  | 'CUSTOM_API_REQUEST_STARTED'
+  | 'DOCUMENT_CAMERA_ERROR'
+  | 'DOCUMENT_CAMERA_ERROR'
+  | 'DOCUMENT_CAMERA_SHUTTER_CLICK'
+  | 'DOCUMENT_CAPTURE'
+  | 'DOCUMENT_CAPTURE'
+  | 'DOCUMENT_CAPTURE'
+  | 'DOCUMENT_CAPTURE'
+  | 'DOCUMENT_CONFIRMATION'
+  | 'DOCUMENT_CONFIRMATION'
+  | 'DOCUMENT_FALLBACK_CLICKED'
+  | 'DOCUMENT_IMAGE_QUALITY_GUIDE'
+  | 'DOCUMENT_TYPE_SELECTION'
+  | 'DOCUMENT_VIDEO_CAPTURE'
+  | 'DOCUMENT_VIDEO_CAPTURE'
+  | 'DOCUMENT_VIDEO_FALLBACK_TRIGGERED'
+  | 'FACE_INTRO'
+  | 'FACE_SELFIE_CAMERA_ERROR'
+  | 'FACE_SELFIE_CAPTURE'
+  | 'FACE_SELFIE_CAPTURE'
+  | 'FACE_SELFIE_CONFIRMATION'
+  | 'FACE_SELFIE_FALLBACK_TRIGGERED'
+  | 'FACE_SELFIE_SNAPSHOT_UPLOAD_COMPLETED'
+  | 'FACE_SELFIE_SNAPSHOT_UPLOAD_STARTED'
+  | 'FACE_SELFIE_UPLOAD_STARTED'
+  | 'FACE_VIDEO_CAMERA_ERROR'
+  | 'FACE_VIDEO_CAPTURE'
+  | 'FACE_VIDEO_CAPTURE'
+  | 'FACE_VIDEO_CAPTURE'
+  | 'FACE_VIDEO_CAPTURE'
+  | 'FACE_VIDEO_CHALLENGE_FETCH_ERROR'
+  | 'FACE_VIDEO_CHALLENGE_LOADED'
+  | 'FACE_VIDEO_CHALLENGE_REQUESTED'
+  | 'FACE_VIDEO_CONFIRMATION'
+  | 'FACE_VIDEO_FALLBACK_TRIGGERED'
+  | 'FACE_VIDEO_INTRO'
+  | 'FLOW_COMPLETED'
+  | 'FLOW_STARTED'
+  | 'FORBIDDEN_CLIENT_ERROR'
+  | 'GENERIC_CLIENT_ERROR'
+  | 'INTERRUPTED_FLOW_ERROR'
+  | 'POA_CAPTURE'
+  | 'POA_CAPTURE_POA'
```

Comment:
These 2 above are still a bit confusing. What's the difference between POA_CAPTURE and POA_CAPTURE_POA?

Reply:
`POA_CAPTURE_POA` this event name is an anomaly. I don't know where it's triggered but I know we are sending it to Looker. Once we remove the Woopra integration events name won't be generated dynamically and we will have better visibility over where this event is fired so we will be able to choose the appropriate name for it. Added a comment inside the event map